### ISSUE TITLE
[Keyboard Navigation – BotKit]: “Skip to Main content” link and "Back to Top" link is not defined for the page.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/botkit-website#8]
**User Experience:**
Users who depend on Keyboard are getting affected, if “Skip to Main content” and "Back to Top" link is not defined for the page.

**Test Environment:**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://botkit.ai/
Browser: Edge
Tool Used: Developer tool bar (F 12)

**Repro Steps:**
Step 1: Open the above URL in Edge browser.
Step 2: Navigate to the page using keyboard tab key.
Step 3: Verify whether “Skip to main Content” and "Back to Top" link is defined for the page or not.

**Actual Result:**
“Skip to Main content” and "Back to Top"  links are not defined in the page.

**Expected Result:**
“Skip to Main content” and "Back to Top" links should be defined in the page.
User should able to Skip the header section that's why we need "Skip to main content" link.

**Note**: Same issue repros for all the pages in BotKit application.

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={20bce2aa-9952-4a10-af79-3376bcc8766d}]()

[MAS2.4.1_BotKit_ByPass Blocks.zip](https://github.com/howdyai/botkit-website/files/2971028/MAS2.4.1_BotKit_ByPass.Blocks.zip)

**Comment by benbrown** [comment]
Done.

**Comment by v-gasikk** [comment]
Issue is Partially Fixed.
"Back to Top" link is not defined in the pages.

**Comment by benbrown** [comment]
@v-gasikk  The back to top link is present at the bottom of every page, as seen in screenshot below.

Is this incorrectly implemented?

![Screen Shot 2019-03-27 at 9 33 00 AM](https://user-images.githubusercontent.com/729873/55084685-6b1c7900-5073-11e9-8c1e-71c689d2d686.png)

**Comment by v-gasikk** [comment]
Verified. Issue is Fixed. Hence Closing the bug
